src/smart-components/role/add-role-new/add-permissions.js

```diff
@@ -293,6 +308,23 @@ const AddPermissionsTable = ({ selectedPermissions, setSelectedPermissions, ...p
           },
         ]}
         isFilterable={true}
+        rowWrapper={({ row, ...props }) =>
```

Comment:
Can you add this to custom component? If this would have been something small I would ok with it being inline, but since you added tooltip and such I'd much more happy if this would have been in special file.